### ISSUE TITLE
ckeditor not routed ?

### ISSUE THREAD

**Issue by anthonydahanne** [jipiboily/monologue#236]
Hello all,
I've successfully configured monologue in my rails app with one tiny exception ... I can't attach pictures to my posts ...
So the ckeditor seems to be working fine : 
![screen shot 2015-02-05 at 11 24 56](https://cloud.githubusercontent.com/assets/60938/6064160/2dda85be-ad2a-11e4-8a9c-f2ef63d1b958.png)

Except when I try to insert a picture : 
![screen shot 2015-02-05 at 11 25 41](https://cloud.githubusercontent.com/assets/60938/6064164/36301fc6-ad2a-11e4-9026-c59cac315bd9.png)

Well, in fact if ckeditor is being used in my Gemfile.lock, 
```
monologue (0.4.1)
      bcrypt (~> 3.1.7)
      ckeditor (~> 4.0.11)
      coffee-rails (~> 4.0.0)
      jquery-rails
      rails (>= 4.0.4)
      sass-rails (~> 4.0.0)
      select2-rails (~> 3.2)
      truncate_html
```

No route is mentioning it when I do a rake routes (whether in my app routes or monologue routes)

```
Routes for Monologue::Engine:
              root GET      /                                      monologue/posts#index
        posts_page GET      /page/:page(.:format)                  monologue/posts#index
              feed GET      /feed(.:format)                        monologue/posts#feed {:format=>:rss}
         tags_page GET      /tags/:tag(.:format)                   monologue/tags#show
             admin GET      /monologue(.:format)                   monologue/admin/posts#index
      admin_logout GET      /monologue/logout(.:format)            monologue/admin/sessions#destroy
       admin_login GET      /monologue/login(.:format)             monologue/admin/sessions#new
    admin_sessions GET      /monologue/sessions(.:format)          monologue/admin/sessions#index
                   POST     /monologue/sessions(.:format)          monologue/admin/sessions#create
 new_admin_session GET      /monologue/sessions/new(.:format)      monologue/admin/sessions#new
edit_admin_session GET      /monologue/sessions/:id/edit(.:format) monologue/admin/sessions#edit
     admin_session GET      /monologue/sessions/:id(.:format)      monologue/admin/sessions#show
                   PATCH    /monologue/sessions/:id(.:format)      monologue/admin/sessions#update
                   PUT      /monologue/sessions/:id(.:format)      monologue/admin/sessions#update
                   DELETE   /monologue/sessions/:id(.:format)      monologue/admin/sessions#destroy
       admin_posts GET      /monologue/posts(.:format)             monologue/admin/posts#index
                   POST     /monologue/posts(.:format)             monologue/admin/posts#create
    new_admin_post GET      /monologue/posts/new(.:format)         monologue/admin/posts#new
   edit_admin_post GET      /monologue/posts/:id/edit(.:format)    monologue/admin/posts#edit
        admin_post GET      /monologue/posts/:id(.:format)         monologue/admin/posts#show
                   PATCH    /monologue/posts/:id(.:format)         monologue/admin/posts#update
                   PUT      /monologue/posts/:id(.:format)         monologue/admin/posts#update
                   DELETE   /monologue/posts/:id(.:format)         monologue/admin/posts#destroy
       admin_users GET      /monologue/users(.:format)             monologue/admin/users#index
                   POST     /monologue/users(.:format)             monologue/admin/users#create
    new_admin_user GET      /monologue/users/new(.:format)         monologue/admin/users#new
   edit_admin_user GET      /monologue/users/:id/edit(.:format)    monologue/admin/users#edit
        admin_user GET      /monologue/users/:id(.:format)         monologue/admin/users#show
                   PATCH    /monologue/users/:id(.:format)         monologue/admin/users#update
                   PUT      /monologue/users/:id(.:format)         monologue/admin/users#update
                   DELETE   /monologue/users/:id(.:format)         monologue/admin/users#destroy
    admin_comments GET      /monologue/comments(.:format)          monologue/admin/comments#show
admin_post_preview PUT|POST /monologue/post/preview(.:format)      monologue/admin/posts#preview
              post GET      /*post_url(.:format)                   monologue/posts#show
```

Thanks a lot for your help !
Anthony

**Comment by msevestre** [comment]
Salut Anthony,

Please look at the extension that I wrote that enable picture upload

https://github.com/msevestre/monologue_image_upload

A+
Michael

**Comment by anthonydahanne** [comment]
Allo Michael,
Thanks for the link - since my rails app is using paperclip 4.2.1, I had to tweak it a bit to make it work.( I had  conflict between your dependency on paperclip 3+ and mine on 4+)

Here's the pull request in case you're interested : 
https://github.com/msevestre/monologue_image_upload/pull/4

Have you considered pushing this gem of yours to rubygems.org ? it could ease the consumption of your nice gem

Thanks again,
Cheers from Montreal

**Comment by anthonydahanne** [comment]
for people interested, you can consume my version with : 
```ruby
gem "monologue_image_upload", :git => "git@github.com:anthonydahanne/monologue_image_upload.git", :branch => "using-paperclip-4.2.1"

It works fine with 4.1.9, monologue 0.4.1 and paperclip 4.2.1

**Comment by yuya4** [comment]
I faced the same issue where I installed Rails 4.2.0. I tried to downgrade Rails as below, but failed.

Bundler could not find compatible versions for gem "rails":
  In Gemfile:
    monologue_image_upload (>= 0) ruby depends on
      rails (< 4.2.0, >= 4.0.0) ruby

    rails (4.2.0)

And then, with any version of rails this error came,

Bundler could not find compatible versions for gem "railties":
  In Gemfile:
    jquery-rails (>= 0) ruby depends on
      railties (>= 4.2.0) ruby

    rails (= 4.1.9) ruby depends on
      railties (4.1.9)

Please advise me what I can do for troubleshooting.

**Comment by msevestre** [comment]
This should be fixed now as master is compatible with rails 4.2.x
